src/Tribe/Views/V2/Assets.php

```diff
@@ -166,6 +180,24 @@ public function register() {
 				'priority' => 10,
 			]
 		);
+
+		tribe_asset(
+			$plugin,
+			'tribe-events-views-v2-datepicker',
+			'views/datepicker.js',
+			[ 'jquery', 'tribe-common', 'tribe-events-views-v2-bootstrap-datepicker' ],
+			null,
+			[
+				'priority' => 10,
+			]
+		);
+
+		/**
+		 * @todo: remove once we can not load v1 scripts in v2
```

Comment:
:warning: Comment refers to a TODO task

<sub>hash: bd0daab6fccfd12904b60d29985a07d2</sub>